src/components/Btn/Btn.jsx

```diff
@@ -0,0 +1,13 @@
+import { Button } from "@mui/material";
+
+import "./Btn.css";
+
+function Btn({ children, onClick, className = "", ...props }) {
+  return (
+    <Button onClick={onClick} className={`btn ${className}`} {...props}>
```

Comment:
if u don't change any method(f.e. `onClick`) then dont put it as `onClick={onClick}`. It will be inside `props`